scipy/stats/tests/test_distributions.py

```diff
@@ -1374,6 +1374,56 @@ def test_stats2(self):
         assert_equal((m, s), (0., 0.))
         assert_allclose((v, k), (4., 3.25))
 
+    @pytest.mark.parametrize("rvs_loc", [-5, 0, 1, 2])
+    @pytest.mark.parametrize("rvs_scale", [1, 2, 3, 10])
+    def test_fit(self, rvs_loc, rvs_scale):
```

Comment:
Fix the RNG seed.

Reply:
I see 
``` 
def setup_method(self):
        np.random.seed(1234)
``` 
inside the `TestDLaplace` class, should I set the seed inside each method?

Reply:
Yes - well, at least if there is something random going on, like `rvs`. Also, check at the top of the document. If it says `from numpy.random import seed` or something, just use `seed(1234)`. (I remembar Ilayn asking us for that on the LAPACK work.)

Reply:
I'm sorry @swallan. Somehow I didn't read the bit about `setup_method`... it's been a really busy week. It looks like `setup_method` runs before the method runs, you're absolutely right, the seed does not need to be set every time. Sorry!

Reply:
No problem @mdhaber, fixed!